cfme/services/myservice/ui.py

```diff
@@ -179,10 +179,13 @@ class ReconfigureServiceView(SetOwnershipForm):
 
     @property
     def is_displayed(self):
+        # todo check whether this is Feature or Bug (it seems we need to pass parent item instead)
```

Comment:
I think this is 1658906, please include in the comment or set name based on the BZ state

Reply:
I forgot that my PR hadn't been merged and asked @niyazRedhat to create another PR. 
I'll fix this here.